extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -203,8 +231,35 @@ export function getConnectionProfile(
 	};
 }
 
-export async function collectSourceDatabaseTableInfo(sourceConnectionId: string, sourceDatabase: string): Promise<TableInfo[]> {
-	const ownerUri = await azdata.connection.getUriForConnection(sourceConnectionId);
+export async function getSourceConnectionString(): Promise<string> {
+	return await azdata.connection.getConnectionString((await getSourceConnectionProfile()).connectionId, true);
+}
+
+export async function getTargetConnectionString(
+	serverName: string,
+	azureResourceId: string,
+	username: string,
+	password: string): Promise<string> {
+
+	const connectionProfile = getTargetConnectionProfile(
+		serverName,
+		azureResourceId,
+		username,
+		password,
+		// for login migration, when connecting to a target Azure SQL, use true/true
+		true /* encryptConnection */,
+		true /* trustServerCertificate */);
```

Comment:
I don't think trust=true is correct, we should not trust cert connections to azure?